frontend/src/pages/Projects.tsx

```diff
@@ -61,7 +61,7 @@ const ProjectsPage = () => {
       empty="No projects found"
       searchPlaceholder="Search for OWASP projects..."
     >
-      {projects && projects.map(renderProjectCard)}
+      {projects && projects.filter((project) => project.is_active).map(renderProjectCard)}
```

Comment:
We don't need inactive projects to be returned from the index, the filtering should be done on the API request level. 

We can improve it later.